Burricos stack trace

2015-01-06 20:42:47 [ERROR] Could not pass event InventoryClickEvent to Burricos v1.2                                                 │·········································
org.bukkit.event.EventException                                                                                                       │·········································
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9│·········································
b]                                                                                                                                    │·········································
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b$│·········································
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec$│·········································
b]                                                                                                                                    │·········································
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec$│·········································
b]                                                                                                                                    │·········································
        at net.minecraft.server.v1_7_R4.PlayerConnection.a(PlayerConnection.java:1431) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b$│·········································
        at net.minecraft.server.v1_7_R4.PacketPlayInWindowClick.a(SourceFile:32) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]      │·········································
        at net.minecraft.server.v1_7_R4.PacketPlayInWindowClick.handle(SourceFile:10) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b] │·········································
        at net.minecraft.server.v1_7_R4.NetworkManager.a(NetworkManager.java:184) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]     │·········································
        at net.minecraft.server.v1_7_R4.ServerConnection.c(ServerConnection.java:81) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]  │·········································
        at net.minecraft.server.v1_7_R4.MinecraftServer.v(MinecraftServer.java:734) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]   │·········································
        at net.minecraft.server.v1_7_R4.DedicatedServer.v(DedicatedServer.java:289) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]   │·········································
        at net.minecraft.server.v1_7_R4.MinecraftServer.u(MinecraftServer.java:584) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]   │·········································
        at net.minecraft.server.v1_7_R4.MinecraftServer.run(MinecraftServer.java:490) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b] │·········································
        at net.minecraft.server.v1_7_R4.ThreadServerApplication.run(SourceFile:628) [spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec9b]   │·········································
Caused by: java.lang.NullPointerException                                                                                             │·········································
        at com.gipsyking.Burricos.Burricos.isZipItem(Burricos.java:317) ~[?:?]                                                        │·········································
        at com.gipsyking.Burricos.Burricos.inventoryClickEvent(Burricos.java:153) ~[?:?]                                              │·········································
        at sun.reflect.GeneratedMethodAccessor193.invoke(Unknown Source) ~[?:?]                                                       │·········································
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.7.0_65]                        │·········································
        at java.lang.reflect.Method.invoke(Method.java:606) ~[?:1.7.0_65]                                                             │·········································
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:298) ~[spigot.jar:git-Spigot-1.7.9-R0.2-203-g02dec$│·········································
b]                                                                                                                                    │·········································
        ... 13 more

--- a/plugin.yml
+++ b/plugin.yml
@@ -1,5 +1,5 @@
 name: Burricos
 main: com.gipsyking.Burricos.Burricos
-version: 1.3
+version: 1.4
 description: Allows larger donkey inventories when using a crate item with special lore "Donkey double chest"
 author: gipsyking

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
   <groupId>com.gipsyking.Burricos</groupId>
   <artifactId>Burricos</artifactId>
   <packaging>jar</packaging>
-  <version>1.3-SNAPSHOT</version>
+  <version>1.4-SNAPSHOT</version>
   <name>Burricos</name>
   <url>https://github.com/gipsy-king/Burricos</url>
    

--- a/src/com/gipsyking/Burricos/Burricos.java
+++ b/src/com/gipsyking/Burricos/Burricos.java
@@ -151,7 +151,7 @@ public class Burricos extends JavaPlugin implements Listener{
 		// now it is an extended donkey inventory
 		
 		ItemStack item = event.getCurrentItem();
-		if (isZipItem(item)) {
+		if (item != null && isZipItem(item)) {
 			event.setCancelled(true);
 			return;
 		}

